src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -520,7 +519,14 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
       auto sbcip = std::make_unique<ScillaBCInfo>(
           m_curBlockNum, m_curDSBlockNum, m_originAddr, contractAddress,
           contractAccount->GetStorageRoot(), scilla_version);
-      m_scillaIPCServer->setBCInfoProvider(std::move(sbcip));
+
+      if (m_scillaIPCServer) {
```

Comment:
At this moment I realized the relationship is too complicated between AccountStoreSC and `m_scillaIPCServer`, which can indeed be nullptr, and even if not, it could probably not be started/listening.  Probably need to simplify this later on.  Not blocking submission of course.